Zotero/Scenes/Detail/PDF/ViewModels/PDFReaderActionHandler.swift

```diff
@@ -1369,7 +1369,9 @@ final class PDFReaderActionHandler: ViewModelActionHandler, BackgroundDbProcessi
         let finalAnnotations = keptAsIs + toAdd
         for annotation in finalAnnotations {
             if annotation.key == nil {
-                annotation.user = viewModel.state.displayName
+                // We use the displayName, but if this is empty we use the username, which is what would be presented anyway.
+                // Since a username cannot be empty, we guarranty an non-empty annotation.user field.
```

Comment:
`guarantee` (or `guaranty`), not `guarranty`

Reply:
Oops, check https://github.com/zotero/zotero-ios/pull/1045/commits/50676c641f1c50bd1b589ba982c950c3ea01900e